buildSrc/src/main/kotlin/org/cqfn/save/generation/Generation.kt

```diff
@@ -82,6 +82,22 @@ fun FunSpec.Builder.generateOptions(jsonObject: Map<String, Option>): FunSpec.Bu
     return this
 }
 
+/**
+ * Adds statement with vararg CLI argument for testFiles
+ *
+ * @return builder
+ */
+fun FunSpec.Builder.addTestsVararg(): FunSpec.Builder = apply {
```

Comment:
May be we can create additional `if` in `FunSpec.Builder.generateOptions`, and if `it.key == testFiles`, than manually add 
```
.optional()
.vararg()
```
statements instead of this function? Also, seems you're forget about `fullName` for this option

Reply:
> Also, seems you're forget about fullName for this option
This argument can't be provided by it's name (not sure, if it's bug or feature). Running `save --testFiles foo` just adds `["--testFiles", "foo"]` to these varargs. Also, normally `fullName` is deducted from variable name, we specify it explicitly, because for some options they are different and we wanted to have all generated options consistent.

Reply:
Actually, adding `testFiles` in JSON generally isn't a very good idea, because it will get added to the options table, and we still will need to tweak logic, so that it's representation in the table is reasonable. Let's keep it hard-coded in generation code for now, I think it's enough to write about these parameters in readme (or in `save --help`)